MvvmCross/Windows/Uwp/Views/MvxWindowsMultiRegionViewPresenter.cs

```diff
@@ -70,14 +70,14 @@ public override void Close(IMvxViewModel viewModel)
             }
         }
 
-        private static Type GetViewType(MvxViewModelRequest request)
+        protected static Type GetViewType(MvxViewModelRequest request)
         {
             var viewFinder = Mvx.Resolve<IMvxViewsContainer>();
             return viewFinder.GetViewType(request.ViewModelType);
         }
 
         // Implementation from: http://stackoverflow.com/a/1759923/80186
-        internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
+        protected internal static T FindChild<T>(DependencyObject reference, string childName) where T : DependencyObject
```

Comment:
I dont know if we understand each other or we have the same vision but translated in a different way. Internal will normally be visible within his namespace. By changing it to protected it will be only visible to itself and subclasses. So to prevent a crash for classes in the same namespace which (can be) used this method, we support the old behaviour and subclasses (in other namespaces). Is this what you also ment to @Cheesebaron?